assets/js/translator.js

```diff
@@ -12,12 +12,13 @@ var UPLOAD_FILE_SIZE_LIMIT = 32E6,
     TRANSLATION_LIST_WIDTH = 650,
     TRANSLATION_LIST_ROWS = 8,
     TRANSLATION_LIST_COLUMNS = 4,
-    TRANSLATION_LISTS_BUFFER = 50;
+    TRANSLATION_LISTS_BUFFER = 50,
+    WEBPAGE_TRANSLATION_BUFFER = 300;
```

Comment:
This really isn't a buffer. I suggest moving up the two constants used later in this file for timeouts and grouping them with this one. Let's also use 500.